langchain/vectorstores/elastic_vector_search.py

```diff
@@ -68,6 +68,20 @@ def __init__(
             )
         self.client = es_client
 
+    def add_texts(self, texts: List[str]) -> None:
+        """Run more texts through the embeddings and add to the vectorstore."""
+        requests = []
+        for i, text in enumerate(texts):
+            request = {
+                "_op_type": "index",
+                "_index": self.index_name,
+                "vector": self.embedding_function(text),
+                "text": text,
+            }
+            requests.append(request)
+        bulk(self.client, requests)
+        self.client.indices.refresh(index=self.index_name)
```

Comment:
This is from memory -- so don't quote me on this, but if I'm not mistaken elasticsearch by default refreshes indices at some set interval (https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#index-refresh-interval-setting)

I think we used to disable refreshes during large bulk writes to speed up the writing

Reply:
it does! I found that when running the ipynb examples, the refresh didn't happen quickly enough to then show up in query results. perhaps another solution is to pass a refresh=True kwarg?

Reply:
yeah that'll work